src/main/webapp/styles/index.css

```diff
@@ -57,10 +57,17 @@
   grid-area: result;
   margin: 4vh 4vw;
 }
+
 #selected-option:hover {
   overflow:auto;
 }
 
+#selected-option textarea {
+  resize: none;
+  width: 600px;
+  height: 350px;
```

Comment:
[Optional] Use vh and vw units instead of px. This way the size of the element adapts to the size of the screen.